docs/use-signalr-service.md

```diff
@@ -92,6 +92,16 @@ By default, all claims from `HttpContext.User` of the negotiate request will be
 They can be accessed at [`Hub.Context.User`](https://github.com/aspnet/SignalR/blob/release/2.2/src/Microsoft.AspNetCore.SignalR.Core/HubCallerContext.cs#L29).
 - Normally you should leave this option as is. Make sure you understand what will happen before customizing it.
 
+<a name="server-sticky-mode"></a>
+
+#### `ServerStickyMode` 
+- Default value is `Disabled`.
```

Comment:
Seems like the default should be preferred no?

Reply:
`Preferred` might have side effect that client connections are not balanced to the app servers, it can have extra burden to the negotiate server, not sure if that is what users want, so we keep it `Disabled` as it is the current behavior.